mesa/visualization/solara_viz.py

```diff
@@ -173,24 +174,23 @@ def ComponentsView(
 
 
 @solara.component
-def ModelController(model: solara.Reactive[Model], play_interval=100):
+def ModelController(
```

Comment:
Awesome work! Maybe more of a clarification question: I previously suggested adding a `*` here to have keyword-only arguments. Maybe I am wrong here and thats why you ignored it, but my understanding was that this is how it works
```python
def foo(bar, baz=2):
    ...
# all of this works:
foo(1)
foo(3, baz=4)
foo(5, 6)
# Compared to
def foo2(bar, *, baz):
    pass

foo2(1) # works
foo2(3, baz=4) # works
foo2(3, 4) # doesnt work
```
My thinking behind this was to have API stability. If we later change the signature to
```python
def ModelController(model, awesome_new_arg, model_parameters=None)
```
we don't have a breaking change. right now it would be breaking if model_parameters is only provided as a positional argument.

Does that make sense? Is that how it works?

Reply:
I was wondering what you ment, but this makes good sense. I'll update it accordingly.

Reply:
Great, feel free to merge afterwards